pkg/chunked/cache_linux_test.go

```diff
@@ -0,0 +1,255 @@
+package chunked
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"reflect"
+	"testing"
+)
+
+const jsonTOC = `
+{
+  "version": 1,
+  "entries": [
+    {
+      "type": "symlink",
+      "name": "bin",
+      "linkName": "usr/bin",
+      "mode": 511,
+      "modtime": "1970-01-01T01:00:00+01:00",
+      "accesstime": "0001-01-01T00:00:00Z",
+      "changetime": "0001-01-01T00:00:00Z"
+    },
+    {
+      "type": "dir",
+      "name": "usr/bin",
+      "mode": 511,
+      "modtime": "2022-01-07T12:36:43+01:00",
+      "accesstime": "0001-01-01T00:00:00Z",
+      "changetime": "0001-01-01T00:00:00Z"
+    },
+    {
+      "type": "reg",
+      "name": "usr/bin/foo",
+      "mode": 511,
+      "size": 103867,
+      "modtime": "1970-01-01T01:00:00+01:00",
+      "accesstime": "0001-01-01T00:00:00Z",
+      "changetime": "0001-01-01T00:00:00Z",
+      "digest": "sha256:99fe908c699dc068438b23e28319cadff1f2153c3043bafb8e83a430bba0a2c6",
+      "offset": 94149,
+      "endOffset": 120135,
+      "chunkSize": 17615,
+      "chunkDigest": "sha256:2ce0d0f8eb2aa93d13007097763e4459c814c8d0e859e5a57465af924169b544"
+    },
+    {
+      "type": "chunk",
+      "name": "usr/bin/foo",
+      "offset": 99939,
+      "chunkSize": 86252,
+      "chunkOffset": 17615,
+      "chunkDigest": "sha256:2a9d3f1b6b37abc8bb35eb8fa98b893a2a2447bcb01184c3bafc8c6b40da099d"
+    }
+}
+`
+
+/*
+func TestHole(t *testing.T) {
```

Comment:
Should this be uncommented.

Reply:
no sorry, this should not be added.  Dropped now